### PR TITLE
Add bare minimum to allow connecting profiler to clio docker

### PR DESCRIPTION
**Pull request by jacarey**
Add jprofiler9 libraries so that we can run jpenable on the container whenever we want to profile clio.

----

### Checklist (never delete this)

Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below.

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: Ensure that you have added or updated tests
- [ ] **Submitter**: If you're adding/switching libraries or otherwise changing the design, update the [design documentation](https://broadinstitute.atlassian.net/wiki/pages/viewpage.action?pageId=114531509).
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
* Review cycle:
  * Team may comment on PR at will
  * **Reviewer assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to reviewer** for further feedback
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Move the JIRA issue to QA once this checklist is completed

### CODE REVIEW

File: project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala [hunk 1]
```diff
@@ -35,7 +35,17 @@ object Docker {
     new Dockerfile {
       from("openjdk:8")
       label("CLIO_VERSION", version.value)
-      expose(8080)
+      runRaw(
+        "wget -q https://download-keycdn.ej-technologies.com/jprofiler/jprofiler_linux_9_2_1.tar.gz -P /tmp/ &&" +
+          " tar -xzf /tmp/jprofiler_linux_9_2_1.tar.gz -C /usr/local &&" +
+          " rm /tmp/jprofiler_linux_9_2_1.tar.gz"
```
Comment (danxmoran) [inline project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala]: Would it be worth sticking `"jprofiler_linux_9_2_1.tar.gz"` in a local val?

File: project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala [hunk 2]
```diff
@@ -35,7 +35,17 @@ object Docker {
...
+      )
+      env(
+        "JPAGENT_PATH",
```
Comment (danxmoran) [inline project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala]: Does this need to be passed to the `java -jar` entry point too?

Comment (jacarey) [reply project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala]: Nope it gets used when you run jpenable in the container

File: project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala (hunk 1 above)
Comment (jacarey) [reply project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala]: Definitely

**Review by danxmoran — APPROVED** [review]

**Review by pshapiro4broad — APPROVED** [review]

**Pull request merged**